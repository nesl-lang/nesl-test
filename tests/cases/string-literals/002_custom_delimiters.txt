%%%[simple]%%%
%%%[empty]%%%
%%%[contains ]%%% in middle]%%%
prefix %%%[with prefix]%%%
%%%[with suffix]%%% suffix
%%%[first]%%% %%%[second]%%%
%%%[unterminated
%%%[valid]%%% extra
no markers