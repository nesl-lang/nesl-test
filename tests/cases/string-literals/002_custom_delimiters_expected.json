{
  "config": {
    "stringOpen": "%%%[",
    "stringClose": "]%%%"
  },
  "results": [
    {"value": "simple"},
    {"value": "empty"},
    {"value": "contains ]%%% in middle"},
    {"error": "string_not_found"},
    {"error": "content_after_string"},
    {"value": "first]%%% %%%[second"},
    {"error": "string_unterminated"},
    {"error": "content_after_string"},
    {"error": "string_not_found"}
  ]
}